e2e/support/commands.js

```diff
@@ -48,3 +48,48 @@ Cypress.Commands.overwrite("visit", (visit, url, options = {}) => {
   }
   return visit(url, options);
 });
+
+/**
+ * The DOMparser.
+ */
+const parser = new DOMParser();
+
+/**
+ * Appends children nodes to the target.
+ *
+ * @param children - A list of DOM nodes.
+ * @param target - The target to append the children to.
+ */
+const apendChildrenTo = (children, target) => {
+  // Clear out the previous content
+  target.innerHTML = "";
+
+  // Append each elment
+  children.forEach((element) => {
+    // Do not append `<script>` tags with `src` defined
+    if (element.tagName !== "script" || element.hasAttribute("src")) {
+      target.appendChild(element);
+    }
+  });
+};
+
+/**
+ * Parses the given text as html and returns an attached to the dom, node reference.
+ *
+ * @param text - The text value to pe parsed into html.
+ * @returns The document element resulted.
+ */
+const parseHTML = (text) => {
+  const doc = parser.parseFromString(text, "text/html");
+  const parentDocument = cy.state("window").parent.document;
+  const iframe = parentDocument.querySelector(".iframes-container iframe");
+
+  apendChildrenTo(Array.from(doc.head.childNodes), iframe.contentDocument.head);
+  apendChildrenTo(Array.from(doc.body.childNodes), iframe.contentDocument.body);
+
+  return iframe.contentDocument;
+};
+
+Cypress.Commands.add("visitSSR", (url) => {
```

Comment:
I just miss a comment here. :eyes:

Reply:
@DAreRodz I'll follow-up with a new PR to not block the 3xx redirections anymore.